docs/pages/example/external-geojson.html

```diff
@@ -0,0 +1,34 @@
+<div id="map"></div>
+
+<script>
+    var map = new mapboxgl.Map({
+        container: 'map', // container ID
+        style: 'mapbox://styles/mapbox/satellite-v9', // style URL
+        zoom: 7, // starting zoom
+        center: [138.043, 35.201] // starting center
+    });
+
+    // Define a variable for external URL string.
+    var url = 'https://docs.mapbox.com/mapbox-gl-js/assets/earthquakes.geojson';
+
+    map.on('load', function () {
+        // Use the variable for the 'data' value
+        // when you add the source to the map.
+        map.addSource('earthquakes', {
+            type: 'geojson',
+            data: url
+        });
```

Comment:
If you do swap this, you could do something like this to retain the helpful comment from above:

```suggestion
            type: 'geojson',
            // Define a variable for external URL string.
            data: 'https://docs.mapbox.com/mapbox-gl-js/assets/earthquakes.geojson'
        });
```

Reply:
Yes, def! Change made in 6753fef